packages/front-end/components/HealthTab/TrafficCard.tsx

```diff
@@ -151,11 +151,10 @@ export default function TrafficCard({
           {!selectedDimension && (
             <div className="ml-auto">
               Cumulative{" "}
-              <Toggle
-                label="Cumulative"
-                id="cumulative"
-                value={cumulative}
-                setValue={setCumulative}
+              <Switch
+                id="cumulativeTrafficSwitch"
```

Comment:
```suggestion
```

I'd probably remove the `id` since there will be duplicate ids due to the dashboard using the same TrafficCard